[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - LOGS_CAPTURED - Started capturing logs in process (pid: 16465).
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - Merged_data - STEP_START - Started execution of step "Merged_data".
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - Merged_data - Loading file from: /Users/nirmalgeorge/dap_dagster_project/DAP-project/govt_employee_analysis/tmpjy9nrer1/storage/JsonLoading using PickledObjectFilesystemIOManager...
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - Merged_data - ASSET_OBSERVATION - DagsterEventType.ASSET_OBSERVATION for step Merged_data
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - Merged_data - LOADED_INPUT - Loaded input "JsonLoading" using input manager "io_manager"
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - Merged_data - Loading file from: /Users/nirmalgeorge/dap_dagster_project/DAP-project/govt_employee_analysis/tmpjy9nrer1/storage/LoadingCsv using PickledObjectFilesystemIOManager...
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - Merged_data - ASSET_OBSERVATION - DagsterEventType.ASSET_OBSERVATION for step Merged_data
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - Merged_data - LOADED_INPUT - Loaded input "LoadingCsv" using input manager "io_manager"
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - Merged_data - STEP_INPUT - Got input "JsonLoading" of type "Any". (Type check passed).
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - Merged_data - STEP_INPUT - Got input "LoadingCsv" of type "Any". (Type check passed).
[32m2023-12-18 18:45:51 +0000[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - 56473f41-daa1-49f5-bc4a-e1293c654bd0 - 16465 - Merged_data - STEP_FAILURE - Execution of step "Merged_data" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "Merged_data"::

KeyError: 'name'

Stack Trace:
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/dagster/_utils/__init__.py", line 465, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 131, in _coerce_op_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 125, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/dap_dagster_project/DAP-project/govt_employee_analysis/govt_employee_analysis/assets.py", line 178, in Merged_data
    data_df = pd.merge(reimburesment_df,employee_df,on=['name','department'], how = 'inner')
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/pandas/core/reshape/merge.py", line 110, in merge
    op = _MergeOperation(
         ^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/pandas/core/reshape/merge.py", line 703, in __init__
    ) = self._get_merge_keys()
        ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/pandas/core/reshape/merge.py", line 1179, in _get_merge_keys
    left_keys.append(left._get_label_or_level_values(lk))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/nirmalgeorge/anaconda3/lib/python3.11/site-packages/pandas/core/generic.py", line 1850, in _get_label_or_level_values
    raise KeyError(key)
[0m
